advanced_filters/views.py

```diff
@@ -69,9 +69,8 @@ def get(self, request, model=None, field_name=None):
                 logger.debug('No choices calculated for field %s of type %s',
                              field, type(field))
             else:
-                choices = model_obj.objects.values_list(field.name, flat=True)
-                if choices.count() < max_choices:
-                    choices = set(choices)
+                choices = model_obj.objects.values_list(field.name, flat=True).distinct()
```

Comment:
I think we should add an explicit `order_by` in case the model's default queryset manager doesn't `order_by` a field outside of our values list. This is explained in [`distinct()` docs - first note](https://docs.djangoproject.com/en/1.10/ref/models/querysets/#distinct).